src/docs/taiko.mdx

```diff
@@ -0,0 +1,165 @@
+        name: Discord
+---
+
+<Section title="Overview">
+
+    Taiko is a decentralized, Type 1 ZK-Rollup. The protocol does not have a sequencer allowing anyone to become a proposer in a premissionless manner. Prover markets are implemented allowing anyone to be a prover and earn fees for proving L2 blocks.
+    
+    ###### Focus
+
+    - Full OPCODE EVM equivalence.
+
+
+    <Labels labels={['ZK-Rollup', 'EVM', 'Testnet']} title="Taiko" />
+
+</Section>
+
+<Section title="General">
+
+    <Parameter name="Block Time" value="On demand*" tooltip="The rate at which the rollup produces blocks. Blocks are produced on demand."/>
+
+    <Parameter name="Gas Limit" value="8.18 million" tooltip="The gas limit that can be consumed by an L2 block. 180 000 gas is reserved for the anchor transaction."/>
+
+    <Parameter name="Finality" value="90 minutes" tooltip="The maximum time it takes for an L2 transaction to be considered final after it is sequenced on L1" />
+
+    <Parameter name="Rollup’s State Contract on L1" value={<Reference label="0x95fF8D3CE9dcB7455BEB7845143bEA84Fe5C4F6f" url="https://sepolia.etherscan.io/address/0x95fF8D3CE9dcB7455BEB7845143bEA84Fe5C4F6f" />} tooltip="The contract used for sequencing, proving and storing the state of the L2 network" />
+
+    <Parameter name="State Trie" value="Merkle Patricia Trie + Keccak Hashing" tooltip="The data structure used to represent the state of the Rollup along with the hashing algorithm used to compute the root of the trie" />
+
+    <Parameter name="Execution Node Implementations" value={<Reference label="taiko-geth" url="https://github.com/taikoxyz/taiko-geth" />} tooltip="The different execution-client implementations of the rollup" />
+    
+    <Parameter name="Proposer and Prover Implementations" value={<Reference label="taiko-client" url="https://github.com/taikoxyz/taiko-client" />} tooltip="The different client implementations of the rollup" />
+
+    <Parameter name="Transaction Types" tooltip="The types of transactions supported on the Rollup" />
+
+        - **Type 0** - User Transaction. Represent [pre-EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) transactions
+        - **Type 1** - User Transaction. Represent legacy transactions post [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) transactions
+        - **Type 2** - User Transaction. [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) transactions
+    ___
+
+    <MultiRowParameters title="Messaging" tooltip="The Rollup provides native communication with L1. Anyone is able to transmit arbitrary messages and the Taiko protocol will provide proof of the data transfer on the destination chain" data={[
+        {
+            title: 'L1 → L2',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: 'The maximum time it takes for a message to be made available on the rollup after it is included in the Ethereum block.',
+                    value: '90 minutes',
```

Comment:
Does the rollup really wait for 90 minutes before they import messages? 🤔

Reply:
That's the maximum time constrained by the protocol. Average time is usually <10 minutes, however that is the max so i decided to modify the tooltip instead of giving an estimate that may be wrong in the future